src/types.ts

```diff
@@ -83,3 +83,28 @@ export type Venue = {
 export type SearchResult = {
   venue: Venue;
 };
+
+export type TranslationsObject = {
+  fi: string;
+  en: string;
+  sv: string;
+};
+
+export type VenueDetails = {
+  id: string;
+  data_source: string | null;
```

Comment:
Should we use `camelCase` considering we are within a JS app and are creating an API that will only be used by a JS based app?